docs/xml/elements/party.rst

```diff
@@ -1,35 +1,35 @@
-+---------------------+-----------------------+-----------+----------+----------------------+----------------------------+
-| ExternalIdentifiers | ExternalIdentifiers   | Optional  | Single   |                      |                            |
-+---------------------+-----------------------+-----------+----------+----------------------+----------------------------+
-| LogoUri             | xs:anyURI             | Optional  | Single   |                      |                            |
-+---------------------+-----------------------+-----------+----------+----------------------+----------------------------+
-| Name                | InternationalizedText | Optional  | Single   |                      |                            |
-+---------------------+-----------------------+-----------+----------+----------------------+----------------------------+
-
-Party.Color
------------
-
-+--------+--------------------+--------------+------------+------------------------------+----------------------------+
-| Tag    | Data Type          | Required?    | Repeats?   |Description                   |Error Handling              |
-|        |                    |              |            |                              |                            |
-+========+====================+==============+============+==============================+============================+
-|        |                    |              |            |                              |                            |
-|        |                    |              |            |                              |                            |
-+--------+--------------------+--------------+------------+------------------------------+----------------------------+
++---------------------+---------------------------+-----------+----------+----------------------+-------------------------------+
+| Tag                 | Data Type                 | Required? | Repeats? |Description           |Error Handling                 |
+|                     |                           |           |          |                      |                               |
++=====================+===========================+===========+==========+======================+===============================+
+| Abbreviation        | xs:string                 | Optional  | Single   |An abbreviation for   |If the field is invalid or not |
+|                     |                           |           |          |the party name.       |present, the implementation is |
+|                     |                           |           |          |                      |required to ignore it.         |
++---------------------+---------------------------+-----------+----------+----------------------+-------------------------------+
+| Color               | HtmlColorString           | Optional  | Single   |The preferred display |If the field is invalid or not |
```

Comment:
No. Not sure we do `TimeWithZone`, either. We should probably fix that. I'll file issues.